source/controller/project_controller.py

```diff
@@ -179,8 +178,6 @@ def handle_tree_double_click(self):
         if type_item == "g6 File" or type_item == "txt File":
             with open(file_path) as file:
                 graph = file.read().splitlines()
-                print(type(graph))
-                print(graph)
```

Comment:
@atilaajones But in this commit, the prints were removed, weren't they?

Reply:
You are right, sorry.